game_functions/task_function.py

```diff
@@ -7,7 +7,7 @@ class task_functions:
     def __init__(self, win):
         self.trialClock=core.Clock()
         image_path = 'Images/Tasks/'
-        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], color=u'white',text=u'+')
+        self.fixation=visual.TextStim(win, ori=0, font=u'Arial', pos=[0, 0], height=50, color=u'white',text=u'+')
```

Comment:
I did that before but the history becomes very complicated and the push is also very messy in both the gui the website. So I deleted my additional branch. But how do you exclude a commit from a PR?